README.rst

```diff
@@ -70,21 +70,27 @@ are contained in brackets.
-+-----------------------+---------------------------------------------------------------------------------------------+
-| ``[`` *field* ``]``   | Same as *field*                                                                             |
-+-----------------------+---------------------------------------------------------------------------------------------+
-| ``[`` *idx* ``]``     | Array access, described below (this is always unambiguous with field access)                |
-+-----------------------+---------------------------------------------------------------------------------------------+
++---------------------------------+---------------------------------------------------------------------------------------------+
+| Syntax                          | Meaning                                                                                     |
++=================================+=============================================================================================+
+| ``$``                           | The root object                                                                             |
++---------------------------------+---------------------------------------------------------------------------------------------+
+| ```this```                      | The "current" object.                                                                       |
++---------------------------------+---------------------------------------------------------------------------------------------+
+| ```foo```                       | More generally, this syntax allows "named operators" to extend JSONPath is arbitrary ways   |
++---------------------------------+---------------------------------------------------------------------------------------------+
+| *field*                         | Specified field(s), described below                                                         |
++---------------------------------+---------------------------------------------------------------------------------------------+
+| ``[`` *field* ``]``             | Same as *field*                                                                             |
++---------------------------------+---------------------------------------------------------------------------------------------+
+| ``[`` *idx* ``]``               | Array access by index, described below (this is always unambiguous with field access)       |
++---------------------------------+---------------------------------------------------------------------------------------------+
+| ``[`` ?*filter* ``]``           | Array access with filter, described below                                                   |
++---------------------------------+---------------------------------------------------------------------------------------------+
+| ``[``\ ``\\`` *jsonpath*\ ``]`` | Ascendant sorted version of an array (may be comma-separated list)                          |
```

Comment:
Ascending. And spell out that it is *jsonpath*`,` *jsonpath*`,` ... for the comma separated list. If I write *jsonpath* [ `[/` *jsonpath* `]` does it return all the matches with each sorted, or does it combine them and then sort that?

Please make it very clear that that sort is Python comparison. Because this is the very first part of the library that is Python-specific. All the rest of the functions are language-independent. This library is used in ways where the user has no idea there is any Python underneath.